packages/datagrid/src/components/DataGrid/DataGrid.component.js

```diff
@@ -288,6 +296,13 @@ export default class DataGrid extends React.Component {
 		return agGridOptions;
 	}
 
+	shouldUpdateCurrentColumn() {
```

Comment:
i was sure this change has already been commited

Reply:
I was up to date with master and eslint was yelling at me :)